### DISCUSSION TITLE
FPS limiter?

### DISCUSSION THREAD

**Discussion by GajuszZOrod** [Try/OpenGothic#340] [Ideas]
Hi,

Is there a way to limit the FPS?
There are places on the map (few steps from Xardas' tower) where I have 230-280 fps, and my GPU (RTX 2070) shows ~75% usage.
Then if I enter the game menu (game gets paused) FPS rises to 344 and my GPU shows 99% usage.
Such high FPS is pointless as my monitor does not support such frequencies. Would be nice if we could put a limit on fps.

Regards!

**Comment by thokkat** [comment]
Open `System/Gothic.ini` in your Gothic II install folder and change the entry `zMaxFPS=0` to your desired Fps limit e.g. `zMaxFPS=144`.

**Reply by GajuszZOrod** [reply]
Works :) thank you.
